### DISCUSSION TITLE
Preciso mudar a tela de login do Laravel

### DISCUSSION THREAD

**Discussion by vinix13x** [filamentphp/filament#11726] [Help]
### Package

Panel builder

### Package Version

10

### How can we help you?

Help, preciso mudar a tela de login do laravel e colocar um arquivo tailwing css. como eu faç?

**Comment by leandrocfe** [comment]
P/ mudar a tela de login, você pode estender a classe original e passar no seu painel:

```php
return $panel
    ->login(MyCustomLogin::class)
```
```php
class MyCustomLogin extends \Filament\Pages\Auth\Login
{
//...
}
```

Para customizar um tema, use: https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

**Reply by vinix13x** [reply]
Tem algum video, que possa me dá um norte por favor

**Reply by vinix13x** [reply]
Já revirei a net

**Reply by leandrocfe** [reply]
Nesse video, eu edito a página de profile do Filament. Você pode seguir os mesmos passos pra editar sua página de login.
https://www.youtube.com/watch?v=xRPT0osJUx4

O que vc quer editar no seu login?

Sobre o tema, tenho esses outros 2 videos de customização:

https://www.youtube.com/watch?v=w3yEEgNBn6c
https://www.youtube.com/watch?v=JvBOr_VLCm0

**Reply by vinix13x** [reply]
Agradeço demais Leandro
